src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -28,6 +28,7 @@
   <ItemGroup>
 	<ProjectReference Include="../Microsoft.SqlTools.Hosting/Microsoft.SqlTools.Hosting.csproj" />
 	<ProjectReference Include="../Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj" />
+	<ProjectReference Include="../Microsoft.SqlTools.Serialization/Microsoft.SqlTools.Serialization.csproj" />
```

Comment:
Isn't this the exact wrong way around? Serialization will rely on the ServiceLayer code. The service layer shouldn't rely on that other exe, it's where all the core code is.

Reply:
I included the dependency to serialization here since I was initializing serialization within the host loader for the serialization service. Since I'll be removing that initialization, I can remove service layer's dependency on serialization. Once the dependency on serialization is moved, I can move the contracts I moved to hosting back to the service layer. This was mainly done earlier since trying to access the contracts in the service layer in the serialization project would cause a circular dependency. Since service layer will no longer have a dependency on serialization, then I will be free to import contracts from service layer into serialization.